### ISSUE TITLE
ccmsetup.msi does not exist

### ISSUE THREAD

**Issue by rlsheldon** [MicrosoftDocs/memdocs#1634]
The directions say to upload "ccmsetup.msi". However, that file doesn't exist in my client install directory. I have a client.msi that I can upload from both the i386 and amd64 folders. However, when I try to use that MSI, I get a failure that says the installer was not started by ccmsetup.exe. Either this is a feature that was tested but never added to Configuration Manager or the documentation needs to be updated to reflect the correct file and process.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 875e279b-9029-d1e5-8a6c-0e93ce38ddba
* Version Independent ID: 80bc34d7-3ef7-1c4f-da41-0474b1c5f4a4
* Content: [Co-manage internet-based devices - Configuration Manager](https://docs.microsoft.com/en-us/mem/configmgr/comanage/how-to-prepare-win10)
* Content Source: [memdocs/configmgr/comanage/how-to-prepare-Win10.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/configmgr/comanage/how-to-prepare-Win10.md)
* Product: **configuration-manager**
* Technology: **configmgr-comanage**
* GitHub Login: @mestew
* Microsoft Alias: **mstewart**

**Comment by aczechowski** [comment]
Hi @rlsheldon, I'm sorry you're having problems. As the documentation states:

> Find this file in the following folder on the Configuration Manager site server: `<ConfigMgr installation directory>\bin\i386`.

So look on the _site server_ in the _server installation directory_, not the client install directory.

If you have further questions, I want to make sure it gets to the right people that can help. There are a couple of options available to provide the best level of support for this:
- Post on [Configuration Manager forums](https://docs.microsoft.com/answers/products/mem). These are moderated by our community MVPs who are very experienced, knowledgeable, and helpful.
- Use [Microsoft Support](https://aka.ms/cmcbsupport) to search for the issue, or open a case with professional support.
- Contact your Microsoft Premier team, such as a Premier Field Engineer or Technical Account Manager.

I'm closing this issue as there's nothing actionable for docs at this time.

**Comment by rlsheldon** [comment]
Hi @aczechowski, it seems I was unclear.

You are correct that it does say to look for the media from those locations. I can confirm that from our site server installation that there is no "ccmsetup.msi" file in the directory structure. I have also pulled the latest Configuration Manager baseline install media. That file also does not exist within the standard Configuration Manager install.

So I'm not sure where you are finding the "ccmsetup.msi", but it does not currently exist in current versions of the product. Let me know what additional clarification you need from me.

**Comment by aczechowski** [comment]
Hi @rlsheldon I checked the product build shares where the engineering system creates the installation source files. I also checked the latest tech preview build, here's a screenshot:

![image](https://user-images.githubusercontent.com/26784733/119549041-b01fc100-bd4b-11eb-991c-9de1179d99b5.png)

Note, that's a custom install location, not the default.
If you don't see it in that bin\i386 directory, then something is awry, and I'd suggest engaging with professional support for assistance.
